css/design_system/components/card.scss

```
@@ -0,0 +1,111 @@
+ /**
+ * @title Basic Card
+ * @category components
+ * @component_type card
+ * @value s-card
+ * @description The basic card allows for a title, subtitle, content, and footer - all of which are optional. The following example shows a card that uses all options.
+ *
+ * @example
+ * <div class="s-card">
+ *  <div class="s-card-header">
+ *    <h2 class="s-card-title">Card Title</h2>
+ *    <h3 class="s-card-subtitle">Card Subtitle</h3>
+ *  </div> 
+ *  <div class="s-card-content">
+ *    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum risus odio quis sit mattis in luctus. Dignissim auctor aliquam suspendisse non in ac, purus. Aliquam arcu justo quam lorem at id proin morbi. A arcu accumsan, eu id fringilla non.</p>
+ *  </div>
+ *  <div class="s-card-footer">
+ *    <p>Footer content</p>
```

We can probably just get rid of this `<p>` tag here
```suggestion
 *    Footer content
```